tests/test_docstring_parser.py

```diff
@@ -0,0 +1,86 @@
+from pathlib import Path
+
+from r2t2.docstring_parser import (
+    CodeDocumentComment,
+    iter_extract_docstring_from_lines,
+    iter_extract_docstring_from_file,
+    iter_extract_docstring_from_files
+)
+
+
+DOC_STRING_LINE_1 = 'the docstring line 1'
+DOC_STRING_LINE_2 = 'the docstring line 2'
+
+
+class TestIterExtractDocstringFromLines:
+    def _test_should_extract_no_docstrings_from_empty_file(self):
```

Comment:
Is the leading underscore to skip the test? If so, the canonical way to do this with pytest would be to decorate with `pytest.mark.skip` or `pytest.mark.xfail` depending on whether you expect it to currently fail or not.

Reply:
oops. That was meant to be there only temporarily. I shall fix that in another PR.